### DISCUSSION TITLE
glib.h:No such file or directory

### DISCUSSION THREAD

**Discussion by copypasteonly** [libvips/libvips#2720] [General]
## Bug report

**Describe the bug**
Trying to compile a simple program, and this error appears 
``` In file included from main.c:2:
/usr/include/vips/vips.h:83:10: fatal error: glib.h: No such file or directory
   83 | #include <glib.h>
```
Using -I/usr/include/glib-2.0 the error will be the same but for glibconfig.h instead.

**To Reproduce**
1. Compile
2. See error

**Expected behavior**
Successfully Compile

**Actual behavior**
Error when compiling

**Screenshots**

**Environment**
(please complete the following information)
 - OS: [Arch Linux 5.15.12]
 - Vips: [8.12.2-1]

**Additional context**
I successfully compiled a program yesterday and it just started happening today, no updates done and all.

**Comment by jcupitt** [comment]
Hi, how are you compiling your program?

It's best to use `pkg-config`, for example:

```
gcc -g -Wall my-program.c `pkg-config vips --cflags --libs` 
```

It'll generate all the correct compiler flags for you. If it still doesn't work, you have an install problem.

**Reply by copypasteonly** [reply]
Hello that is exactly how I compile my program, how would I know where I went wrong with the install?
I just used the package from arch for installing libvips.

**Reply by copypasteonly** [reply]
A simple restart fixed it!
